src/containers/BooksList.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Book from '../components/Book';
+import { removeBook } from '../actions/index';
+
+const BooksList = ({ books, removeBook }) => {
+  const handleRemoveBook = book => {
+    removeBook(book);
+  };
+
+  return (
+    <table>
+      <thead>
+        <tr>
+          <td>Book ID</td>
+          <td>Title</td>
+          <td>Category</td>
+        </tr>
+      </thead>
+      <tbody>
+        { books.map(book => (
+          <Book key={book.id} book={book} remove={handleRemoveBook} />
+        ))}
+      </tbody>
+
+    </table>
+  );
+};
+
+BooksList.propTypes = {
+  books: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.number.isRequired,
+      title: PropTypes.string.isRequired,
+      category: PropTypes.string.isRequired,
+    }),
+  ),
+  removeBook: PropTypes.func.isRequired,
+};
+
+BooksList.defaultProps = {
+  books: [],
+};
+
+const MapStateToProps = state => ({
+  books: state.books,
+});
+
+const matchDispatchToProps = dispatch => ({
+  removeBook: book => {
+    dispatch(removeBook(book, book.id));
+  },
+});
```

Comment:
Same here:

1. I guess the name doesn't matter in the end, so you can name it in any way, but it is [usually refered to](https://react-redux.js.org/api/connect#mapdispatchtoprops-object-dispatch-ownprops-object) as `mapDispatchToProps`.

2. The [recommended](https://react-redux.js.org/using-react-redux/connect-mapdispatch#two-forms-of-mapdispatchtoprops) way to use `mapDispatchToProps` is using the _object form_.

3. Do you really need to pass both `book` and `book.id`, I checked your reducer and it is only using the id to find the book that's needed to be removed, and if you pass the `book` it is kind of redundant to pass `book.id` as it is part of book, so maybe just passing `book` or `book.id` is enough.